samples/bluetooth/llpm/src/main.c

```diff
@@ -9,6 +9,12 @@
 #include <sys/printk.h>
 #include <zephyr/types.h>
 
+#if defined(CONFIG_BOARD_NRF52840DONGLE_NRF52840)
+#include <usb/usb_device.h>
+#include <drivers/uart.h>
+#include <init.h>
```

Comment:
You can think of removing init.h from the including files because it has been included by device.h. (e.g. usb_device.h, usb_dc.h, device.h)